tests/unit/exceptions.py

```diff
@@ -0,0 +1,33 @@
+import os
+from unittest import TestCase
+from mock import patch
+
+from dvc.exceptions import StageFileCorruptedError
+
+
+class TestStageFileCorruptedError(TestCase):
+    cwd = "/path"
+
+    @patch.object(os.path, "isabs", return_value=True)
+    @patch.object(os, "getcwd", return_value=cwd)
+    @patch.object(os.path, "relpath")
+    def test_should_include_relative_path_when_dep_in_subdir(
+        self, relpath_patch, _, __
+    ):
+        sub_case_path = os.path.join(self.cwd, "path/to/data")
```

Comment:
Please don't hardcode posix paths. Use os.path.join instead.

Reply:
Test removed, I think there is no need for it after introducing suggestions